### PR TITLE
Add grains exercise

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #236

### CODE REVIEW

File: exercises/practice/grains/grains.coffee [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Grains
+  square: (number) ->
+
+  total: () ->
```
Comment (ErikSchierboom) [inline exercises/practice/grains/grains.coffee:4]: Could these be class methods?

```suggestion
  @square: (number) ->

  @total: () ->
```

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/grains/grains.coffee [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Grains
+  @square: (number) ->
```
Comment (ladokp) [inline exercises/practice/grains/grains.coffee:2]: Why are this "@" signs in front of the method? To pass the tests I had to remove them.

Comment (ladokp) [reply exercises/practice/grains/grains.coffee:2]: It looks like the tests are not synchronized with this change.

Comment (BNAndras) [reply exercises/practice/grains/grains.coffee:2]: Could you elaborate on the tests not being synchronized? configlet isn't seeing any missing.

```
./bin/configlet sync --tests -e grains
Updating cached 'problem-specifications' data...
Checking exercises...
The `grains` exercise has up-to-date tests!
```

Comment (BNAndras) [reply exercises/practice/grains/grains.coffee:2]: > Why are this "@" signs in front of the methods?

That signifies they are  class methods so `Grains.square() and `new Grains.square()` both work. I'll investigate why Erik's change isn't working, but the example passed the CI.

Comment (BNAndras) [reply exercises/practice/grains/grains.coffee:2]: I opened up #249 so we can iterate there.

**Pull request merged**